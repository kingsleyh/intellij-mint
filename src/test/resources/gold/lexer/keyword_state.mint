store A {

  state name : String = ""
  state list : Array(Item) = []

}