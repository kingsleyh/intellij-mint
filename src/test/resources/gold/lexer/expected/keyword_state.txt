store ('store')
WHITE_SPACE (' ')
IDENT ('A')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n\n  ')
state ('state')
WHITE_SPACE (' ')
IDENT ('name')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('String')
WHITE_SPACE (' ')
= ('=')
WHITE_SPACE (' ')
STRING_LITERAL ('""')
WHITE_SPACE ('\n  ')
state ('state')
WHITE_SPACE (' ')
IDENT ('list')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('Array')
( ('(')
IDENT ('Item')
) (')')
WHITE_SPACE (' ')
= ('=')
WHITE_SPACE (' ')
[] ('[]')
WHITE_SPACE ('\n\n')
} ('}')